website/assets/js/codecfuncs.js

```diff
@@ -15,6 +15,15 @@ var metric_units = {
   'encode_cputime': 'Encode CPU time in seconds'
 };
 
+// The GraphLine object is used to get a line and its associated
+// formats together in one piece.
+function GraphLine(title, data, options) {
+  this.title = title;
+  this.data = data;
+  this.options = options;
+}
+
+
 function FillInOneTable(url, id) {
```

Comment:
Oops; here we have a problem that the conventions for Javascript are different than for python. Function names should be fillInOneTable and variableNamesLikeThis. I'm fine turning a blind eye to it as long it's consistent.